app/grandchallenge/algorithms/views.py

```diff
@@ -411,14 +411,22 @@ class AlgorithmResultsList(PermissionListMixin, PaginatedTableListView):
     ]
     order_by = "job__created"
 
-    def get_row_context(self, result, *args, **kwargs):
-        checker = ObjectPermissionChecker(self.request.user)
+    def get_row_context(self, result, *args, checker=None, **kwargs):
+        if not checker:
+            checker = ObjectPermissionChecker(self.request.user)
```

Comment:
Just to double check, is this line ever actually hit? If so, could there be a better way to persist the checker that has had `prefetch_perms` applied?

Reply:
It's not, I just included it because the default for the kwarg is None.

Reply:
I actually tested it before adding the if statement and it worked fine

Reply:
Ok, great! Then maybe make it a required kwarg? `checker=None,` -> `checker,`, then you can remove the if block.